Jenkinsfile.release

```diff
@@ -85,9 +85,21 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             cosa init --branch ${params.STREAM} https://github.com/coreos/fedora-coreos-config
             cosa buildfetch --build=${params.VERSION} \
                 --arch=all --url=s3://${s3_stream_dir}/builds
+            cosa buildprep --ostree --build=${params.VERSION} \
+                --arch=all s3://${s3_stream_dir}/builds
             """)
         }
 
+        stage('Sync oscontainer to quay.io') {
+            withCredentials([string(credentialsId: "oscontainer-secret", variable: "OSCONTAINER_SECRET")]) {
+                shwrap("""ociarchive=\$(cosa meta --image-path ostree)
+                        case \${ociarchive} in
```

Comment:
All streams use ociarchive now and at some point we'll probably rip out the old support for .tar in cosa. So I think we can just simplify this logic here.